doc/tutorials/test-templates-basic.md

```diff
@@ -0,0 +1,306 @@
+# Creating Tests with JSonnet Templates
+
+## Prerequisites
+
+1. A clone of this repository.
+1. Build dependencies from [developing.md](/doc/developing.md).
+1. A GKE cluster with GPUs and (optionally) Cloud TPUs. Accelerator availability depends on GCE zone. All of the accelerators used in this tutorial are available in `us-central1-b`.
+  - Example command:
+    ```
+      gcloud beta container clusters create tutorial-cluster \
```

Comment:
I think we should either add an explicit `--project` flag or make a very visible note that this command will use the current default project in the user's shell

Reply:
Done.